src/components/Header/Header.css

```diff
@@ -1,24 +1,25 @@
 .header {
   background-color: #000;
   height: 150px;
-  padding: 100px;
+  padding-top: 100px;
```

Comment:
you can combine these with shorthand! `padding: 100px 0;` The shorthand takes up to 4 options, one for each side (set clockwise, from the top). If you only give 2, it uses the 1st as top and bottom and the 2nd as left and right.